IMAGE:
  WIDTH: 640
  HEIGHT: 480
  # find your ROI parameter.

XYCAR:
  START_SPEED: 0.0
  MAX_SPEED: 0.0
  MIN_SPEED: 0.0
  SPEED_CONTROL_THRESHOLD: 0.0
  ACCELERATION_STEP: 0.00
  DECELERATION_STEP: 0.0
  # find your parameter.

# If you want to use other control methods, Change PID to another control methods.
PID:
  P_GAIN: 0.0
  I_GAIN: 0.0000
  D_GAIN: 0.00
  # find your parameter.

MOVING_AVERAGE_FILTER:
  SAMPLE_SIZE: 30

TOPIC:
  PUB_NAME: /xycar_motor
  SUB_NAME: /usb_cam/image_raw/
  LIDAR_NAME: /scan
  QUEUE_SIZE: 1

DEBUG: true

CAMERA:
  CAMERA_MATRIX: [[360.77831, 0.0, 332.39996],
                  [0.0, 362.68611, 211.57494],
                  [0.0, 0.0, 1.0]]
  DIST_COEFF: [-0.318694, 0.088588, -0.000184, -0.003607, 0.0]
